diff --color -r -c C/testcasesupport/io.c C_patch/testcasesupport/io.c
*** C/testcasesupport/io.c	2017-08-31 21:42:02.000000000 +0800
--- C_patch/testcasesupport/io.c	2021-05-10 09:07:57.327028040 +0800
***************
*** 157,163 ****
  
  int globalReturnsTrueOrFalse() 
  {
!     return (rand() % 2);
  }
  
  /* The variables below are declared "const", so a tool should
--- 157,163 ----
  
  int globalReturnsTrueOrFalse() 
  {
!     return 1;
  }
  
  /* The variables below are declared "const", so a tool should
